stan/math/prim/scal/prob/normal_log.hpp

```diff
@@ -30,7 +30,7 @@ namespace math {
  * @tparam T_loc Type of location parameter.
  */
 template <bool propto, typename T_y, typename T_loc, typename T_scale>
-inline auto normal_log(T_y&& y, T_loc&& mu, T_scale&& sigma) {
+inline auto normal_log(T_y&& y, T_loc&& mu, T_scale&& sigma) { // NOLINT
```

Comment:
Could we add this && case to the linter settings or something. This is so annoying.

Reply:
Yeah I'm looking into that. Also editing my stuff above. I need to pull out the strict specialization in operands and partials